MVCCore.Models/RegisterModel.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Text;
+
+namespace MVCCore.Models
+{
+   public class RegisterModel
+    {
+        [Key]
+        public Int32 UserId { get; set; }
+
+        [Display(Name = "User Name")]
+        [MaxLength(50)]
```

Comment:
here also

Reply:
Which one only Display Name and max length is used